client/src/app/components/followings-search/followings-search.component.html

```diff
@@ -0,0 +1,37 @@
+<mat-card class="followings-container">
+  <h3>Followings ({{ followings.length }})</h3>
+  <input
+    class="followings-search"
+    [(ngModel)]="followingSearch"
+    matInput
+    placeholder="Search for followings" />
+  <div class="users-following-container">
+    <ng-container *ngFor="let following of followings">
+      <div class="following-container" *ngIf="isSearchedFollowings(following)">
+        <img
+          class="following-picture"
+          alt="Profile Picture"
+          width="40"
+          height="40"
+          [src]="
+            'https://static.wikia.nocookie.net/bleachuntoldstories/images/2/24/Hitsugaya_toshiro_profile.png'
+          "
+        />
+        <div class="following-info">
+          <span
+            class="following-name"
+            (click)="navigateToFollowingProfile(following.firstName)"
```

Comment:
I know that this will be modified in future because we might have 2 followings with the same first name therefore we cannot use only the first name to define the route/ticker. I suggest adding a TODO comment here.

Reply:
yeah this will be modified when we will do the logic